### PR TITLE
024

### PR DESCRIPTION
**Pull request by oga-sven**

### CODE REVIEW

File: app/Http/Controllers/CompanyController.php [hunk 1]
```diff
@@ -27,19 +27,19 @@ public function index()
     {
         $companies = Company::paginate(9);
 
-        return view('company.index', ['companies' => $companies]);
+        return view('company.index', ['companies' => $companies, 'crum' => 'home']);
     }
 
     /**
      * Show the form for creating a new resource.
      *
```
Comment (Evergetic) [inline app/Http/Controllers/CompanyController.php:35]: Param vergeten

File: app/Http/Controllers/ReviewController.php [hunk 2]
```diff
@@ -29,9 +29,10 @@ public function index()
     public function create(Request $request)
     {
         $companyid = $request->id;
-
-        return view('review.create', ['companyid' => $companyid]);
-    }
+        $companies = Company::where('id', $companyid)->first();
+        // exit($request);
```
Comment (Evergetic) [inline app/Http/Controllers/ReviewController.php]: Random comment

File: app/Http/Controllers/ReviewController.php [hunk 3]
```diff
@@ -29,9 +29,10 @@ public function index()
     public function create(Request $request)
     {
         $companyid = $request->id;
-
-        return view('review.create', ['companyid' => $companyid]);
-    }
+        $companies = Company::where('id', $companyid)->first();
+        // exit($request);  
+        return view('review.create', ['companyid' => $companyid, 'crum' => 'new-review', 'crum2' => $companies]);
+    }//,  'crum' => 'edit-user', 'crum2' => $user
```
Comment (Evergetic) [inline app/Http/Controllers/ReviewController.php]: Random comment

File: routes/breadcrumbs.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+// Breadcrumbs::for('category', function ($trail, $category) {
+//     $trail->parent('blog');
+//     $trail->push($category->title, route('category', $category->id));
+// });
+
+// // Home > Blog > [Category] > [Post]
+// Breadcrumbs::for('post', function ($trail, $post) {
+//     $trail->parent('category', $post->category);
+//     $trail->push($post->title, route('post', $post->id));
+// });
```
Comment (Evergetic) [inline routes/breadcrumbs.php]: Wats deze comments

**Review by Evergetic — CHANGES_REQUESTED** [review]
Check comments

File: app/Http/Controllers/CompanyController.php (hunk 1 above)
Comment (oga-sven) [reply app/Http/Controllers/CompanyController.php:35]: fixed

**Review by oga-sven — COMMENTED** [review]
fixed

**Review by Evergetic — APPROVED** [review]

**Review by KevinStolk — APPROVED** [review]

**Pull request merged**